portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,40 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Thu's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+  <body id="body">
+    <div class="outerBox">
+        <a href="index.html"><img class="banner" src="images/BannerText.png"/></a>
+        
+        <div class="buttonBox">
+            <a href="academic.html"><img class="button" src="images/Button_Academic.png"/></a>
+            <a href="interests.html"><img class="button" src="images/Button_Interests.png"/></a>
+            <a href="computerSkills.html"><img class="button" src="images/Button_ComputerSkills.png"/></a>
+            <a href="gallery.html"><img class="button" src="images/Button_Gallery.png"/></a>
+        </div>
+
+        <h1>Thu's Portfolio</h1>
+
+        <div class="box1">
+            <p>I am a sophmore currently studying Computer Science at the University of Texas. My family origanted from Vietnam and I am the eldest out of 3. </p>
+            <p>When describing myself, I would say that I am an artist at heart. I enjoy creating anything unique—drawings, graphics, videos, stories, codes, etcetera. I am also very observant when it comes to patterns considering I am a visual learner.</p>
+            <p> At times I am a very introverted person who enjoys stories from any medium such as movies, shows, and books. People typically would describe me as a quiet person. They would also say I am attentive to those around others and find my personality charming. I am even described as a person that has a soft-spoken kind of confidence.</p>
```

Comment:
nit: line too long. we general have a practice of max 120 chars per line such that we'll see the full contents without scrolling right.